src/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")
 set(LT_LLVM_INSTALL_DIR "" CACHE PATH "LLVM installation directory")
 list(APPEND CMAKE_PREFIX_PATH "${LT_LLVM_INSTALL_DIR}/lib/cmake/llvm/")
 
-find_package(LLVM 16.0.0 REQUIRED CONFIG)
+find_package(LLVM 17.0.0 REQUIRED CONFIG)
```

Comment:
Should it be 17.0.0? Can you please recheck

Reply:
As far as I understand, this ensures that a minimum of llvm17.0.0 should be used. Future versions of llvm17 also compile with this option. So ,  I think it should be 17.0.0